test/integration/narrow/routes/incident/details-of-the-smell.test.js

```diff
@@ -25,11 +25,11 @@ describe('Experience test', () => {
   })
 
   test('GET /details-of-the-smell route returns 200', async () => {
-    mockSession = {
-      description: 'test description',
-      date: '2017-08-25T00:00:00.000Z',
-      time: '14:35'
-    }
+    // mockSession = {
+    //   date: new Date().toISOString(),
+    //   hour: 14,
+    //   minute: 35
+    // }
```

Comment:
Remove commented out code if not needed?

Reply:
Yes